server/tserver/src/main/java/org/apache/accumulo/tserver/TabletClientHandler.java

```diff
@@ -1281,12 +1282,9 @@ public void compactionJobFailed(TInfo tinfo, TCredentials credentials,
 
   @Override
   public List<String> getActiveLogs(TInfo tinfo, TCredentials credentials) {
```

Comment:
As a follow-on, could this be a List<LogEntry> instead of strings?

Reply:
That would require making LogEntry a Thrift type, as this method is implementing a Thrift interface. I think it's fine to serialize it as a String for the network API.